.github/CONTRIBUTING.md

```diff
@@ -6,12 +6,33 @@
 
 ## Workflow
 
+- `create-post` means the python script `scripts/create-post`, run it using its name (`./scripts/create-post`).
```

Comment:
Yeah, I am thinking about that, but Ayi's command runner is under [a big refactor](https://github.com/dyweb/Ayi/pull/64), and since there is just two command, the advantage of Ayi is not very significant.